telebot/apihelper.py

```diff
@@ -530,7 +530,7 @@ def send_paid_media(
         token, chat_id, star_count, media,
         caption=None, parse_mode=None, caption_entities=None, show_caption_above_media=None,
         disable_notification=None, protect_content=None, reply_parameters=None, reply_markup=None,
-        business_connection_id=None):
+        business_connection_id=None, media_payload=None):
```

Comment:
Why parameter named "media_payload", but not "payload"?

I propose to set it to "payload" to keep naming consistency, and rename internal "payload" to something else. May be "_payload".